packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartBankForm/CounterpartBankForm.test.tsx

```diff
@@ -346,7 +362,28 @@ describe('CounterpartBankForm', () => {
 
         expect(requestCounterpartId).toBe(individualId);
         expect(requestBody).toEqual(serverRequestBody);
-      });
+      }, 10_000);
     });
   });
 });
+
+/**
+ * Check if all counterpart queries are not fetching and do not have an error.
+ *
+ * @param queryClient
+ */
+function checkCounterpartQueriesLoaded(queryClient: QueryClient) {
```

Comment:
I see a drawback here. Right now we check queries but before we checked the `spinner`. From my point of view current approach is less stable because if we add another check to show the bank form the test will fail but before we just wait until the spinner disappears and it doesn't matter which checks we have.

Overall don't get the point to rewrite it into specific query checks

Reply:
In my opinion, the correct test would look like this:
- Test whether a spinner is displayed to show the load status, depending on the `status` of this or that `Query`
- Other tests, the success of which depends on the `status` of this or that `Query`.

Furthermore, if the test crashes because `waitUntilTableIsLoaded(...)` failed to complete successfully, there is no way to know which query is _stuck_ or crashed. With the `checkCounterpartQueriesLoaded(...)` implementation, it is visible _what the problem is_, and _which query is problematic_.